src/components/NodeHostWrapper/NodeHostWrapper.tsx

```diff
@@ -0,0 +1,60 @@
+import block from 'bem-cn-lite';
+
+import {Button, Popover, PopoverBehavior} from '@gravity-ui/uikit';
+
+import type {INodesPreparedEntity} from '../../types/store/nodes';
+import {getDefaultNodePath} from '../../containers/Node/NodePages';
+import {isUnavailableNode, NodeAddress} from '../../utils/nodes';
+
+import EntityStatus from '../EntityStatus/EntityStatus';
+import {NodeEndpointsTooltip} from '../Tooltips/NodeEndpointsTooltip/NodeEndpointsTooltip';
+import {IconWrapper} from '../Icon';
+
+import './NodeHostWrapper.scss';
+
+const b = block('ydb-node-host-wrapper');
+
+interface NodeHostWrapperProps {
+    node: INodesPreparedEntity;
+    getNodeRef?: (node?: NodeAddress) => string;
+}
+
+export const NodeHostWrapper = ({node, getNodeRef}: NodeHostWrapperProps) => {
+    if (!node.Host) {
+        return <span>—</span>;
+    }
+
+    const isNodeAvailable = !isUnavailableNode(node);
```

Comment:
`isNodeAvailable` and everything with it is a new code, everything else - refactoring